src/main/java/com/aws/greengrass/mqttclient/AwsIotMqtt5Client.java

```diff
@@ -71,6 +71,7 @@ class AwsIotMqtt5Client implements IndividualMqttClient {
     private static final String RESUB_LOG_EVENT = "resubscribe";
     private final Provider<AwsIotMqtt5ClientBuilder> builderProvider;
 
+    @Getter // for testing
```

Comment:
Keep it at package visibility if able.

Reply:
done, had synchronize it to appease spotbugs